flow/Makefile

```diff
@@ -98,12 +98,17 @@ ifneq ($(BLOCKS),)
   # Normally this comes from variables.yaml, but we need it here to set up these variables
   # which are part of the DESIGN_CONFIG. BLOCKS is a Makefile specific concept.
   $(foreach block,$(BLOCKS),$(eval BLOCK_LEFS += $(WORK_HOME)/results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}.lef))
-  $(foreach block,$(BLOCKS),$(eval BLOCK_LIBS += $(WORK_HOME)/results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}.lib))
+  $(foreach block,$(BLOCKS),$(eval BLOCK_TYP_LIBS += $(WORK_HOME)/results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}_typ.lib))
+  $(foreach block,$(BLOCKS),$(eval BLOCK_FAST_LIBS += $(WORK_HOME)/results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}_fast.lib))
```

Comment:
I think this is problematic from the point of view that it is the macro config.mk that has to use CORNERS and then generate an abstract with corner .lib files. Makefiles break down here because CORNERS set up in the project defining BLOCKS can be different than CORNERS inside the macro listed in BLOCKS.

Reply:
In other words: if some of the macros in BLOCKS use corners and others don't, then Makefile can't express that here. I don't know if such a thing makes sense in the first place, but it is an example of a more customized flow that a user could need that ORFS won't have adequate testing for.